lib/settingsGenerated.ts

```diff
@@ -62,6 +69,12 @@ export const defaultSettings: ISettings = {
     'preferredLogicTypeSignal': 'unresolved',
     'unusedSignalRegex': '_unused$',
     'ieeeCasing': 'lowercase',
+    'signalPrefix': '',
+    'signalSuffix': '',
+    'genericPrefix': '',
+    'genericSuffix': '',
+    'instantiationLabelPrefix': 'inst_',
```

Comment:
I would either set a often used style guide as default (e.g. `s_` prefix for signal, `g_` prefix for generics and `inst_` prefix for instantiations) or default to no prefixes and suffixes at all.